cyipopt/cython/ipopt_wrapper.pyx

```diff
@@ -160,63 +160,157 @@ cdef class Problem:
+                    Value of the optimization variables at which the Jacobian is
+                    to be evaluated.
+
+                **Returns**
+
+                :py:class:`numpy.ndarray`, shape `(jac_nnz, )`
+                    Jacobian of the constraint functions at the point x.
+
+            - .. py:function:: jacobianstructure()
+
+                Optional callback function that returns the sparsity structure of the
+                Jacobian of the constraints. The function should return a tuple
+                of two :py:class:`numpy.ndarray`\ s of integers,
+                the first array contains the row indices
+                and the second array contains the column indices of the non-zero
+                elements of the Jacobian. Both arrays should have the same
+                length of `jac_nnz`, and indexing is zero-based.
+
+                If this callback is not provided, the Jacobian is assumed
+                to be dense (`jac_nnz` = `m` * `n`) using row-major (C-style) order.
+
+                **Returns**
+
+                :py:class:`tuple`
+                    Tuple containing two numpy arrays, the first array contains
+                    the row indices and the second array contains the column
+                    indices of the non-zero elements of the Jacobian.
+
+            - .. py:function:: hessian(x, lambda, objective_factor)
+
+                Optional callback function for evaluating the Hessian of the Lagrangian
+                function with respect to the primal variables :math:`x`. The Lagrangian
+                is given by
+
+                .. math:: \mathcal{L}(x, \lambda) = \lambda_0 f(x) + \sum_{i=1}^{m} \lambda_i g_i(x)
+
+                where :math:`\lambda_1, \ldots, \lambda_m` are the Lagrange multipliers
+                and :math:`\lambda_0` is the objective factor. The
                 function should return the values of the Hessian as calculated
-                using x, lambda and objective_factor. The values should be
-                returned as a 1-dim numpy array (using the same order as you
-                used when specifying the sparsity structure). If ``None``, the
-                Hessian is calculated numerically.
-            - ``hessianstructure`` : function pointer, optional (default=None)
-                Callback function that accepts no parameters and returns the
-                sparsity structure of the Hessian of the lagrangian (the row
-                and column indices only). If ``None``, the Hessian is assumed
-                to be dense.
+                using `x`, `lambda` and `objective_factor`. The values should be
+                returned as a 1-dimensional array-like structure (using the same order as you
+                used when specifying the sparsity structure).
+
+                If this callback is not provided, the Hessian is calculated numerically.
+
+
+                **Parameters**
```

Comment:
Is it somehow because of the multiple blank lines here?

Reply:
That fixed it for me. Remove one blank line above `**Parameters**`.

Reply:
I removed the line and pushed a commit.

Reply:
Thanks, appreciate it.